### PR TITLE
Amls 5543 - Copying existing content over from amls-frontend

### PR DESCRIPTION
**Pull request by matthewharris1**
This ticket was to make sure the content we currently have in amls-frontend matches the content displayed on the new EAB microservice

Unit and manually tested

### CODE REVIEW

File: app/views/EabServicesProvidedView.scala.html [hunk 1]
```diff
@@ -25,7 +25,7 @@
 @(form: Form[Seq[EabServicesProvided]], mode: Mode)(implicit request: Request[_], messages: Messages)
 
 @main_template(
-    title = s"${errorPrefix(form)} ${messages("eabServicesProvided.title")}"
+    title = s"${errorPrefix(form)} ${messages("eabServicesProvided.title")} - Estate agency business - Manage your anti-money laundering supervision - GOV.UK".trim
```
Comment (slavvco) [inline app/views/EabServicesProvidedView.scala.html]: Could you put "- Estate agency business - Manage your anti-money laundering supervision - GOV.UK" in messages file for me, please? And just use the key? - the same in views below.

File: app/views/PenalisedEstateAgentsActView.scala.html [hunk 2]
```diff
@@ -34,10 +34,20 @@
...
+        <ul class="list list-bullet">
+            <li>@Messages("penalisedEstateAgentsAct.warning")</li>
+            <li>@Messages("penalisedEstateAgentsAct.banned")</li>
+        </ul>
+
         @components.input_yes_no(
             field = form("value"),
             legend = messages("penalisedEstateAgentsAct.heading"),
-            headingIsLegend = true
+            legendClass = Some("visually-hidden")
```
Comment (slavvco) [inline app/views/PenalisedEstateAgentsActView.scala.html]: I think that headingIsLegend = true will be necessary here, we can discuss tomorrow morning.

File: app/views/PenalisedProfessionalBodyView.scala.html [hunk 3]
```diff
@@ -34,10 +34,23 @@
...
+            <li>@Messages("penalisedProfessionalBody.suspension")</li>
+            <li>@Messages("penalisedProfessionalBody.expulsion")</li>
+            <li>@Messages("penalisedProfessionalBody.special")</li>
+        </ul>
+
         @components.input_yes_no(
             field = form("value"),
             legend = messages("penalisedProfessionalBody.heading"),
-            headingIsLegend = true
+            legendClass = Some("visually-hidden")
```
Comment (slavvco) [inline app/views/PenalisedProfessionalBodyView.scala.html]: same as comment above

File: app/views/RedressSchemeView.scala.html [hunk 4]
```diff
@@ -34,10 +34,16 @@
...
+
+        @components.heading("redressScheme.heading")
+
+        <p>@Messages("redressSchemeDetail.content")</p>
+
         @components.input_radio(
             field = form("value"),
             legend = messages("redressScheme.heading"),
-            headingIsLegend = true,
+            legendClass = Some("visually-hidden"),
```
Comment (slavvco) [inline app/views/RedressSchemeView.scala.html]: as above

File: app/views/EabServicesProvidedView.scala.html (hunk 1 above)
Comment (matthewharris1) [reply app/views/EabServicesProvidedView.scala.html]: Done

File: app/views/PenalisedEstateAgentsActView.scala.html (hunk 2 above)
Comment (matthewharris1) [reply app/views/PenalisedEstateAgentsActView.scala.html]: Done ^ also good job adding supportingContent. Fixes issues with tests and content being displayed below header :+1:

**Pull request merged**